packages/central-server/src/database/models/SurveyResponse.js

```diff
@@ -99,7 +99,7 @@ export class SurveyResponseModel extends MaterializedViewLogDatabaseModel {
       [surveyResponseId],
     );
     if (result.length === 0) return false;
-    return result[0].count === '1';
+    return parseInt(result[0].count) === 1;
```

Comment:
@biaoli0 I like the extra safety here in case we pull out that lower level parsing in future
But, my preference is always for "make things blow up in an obvious way, so you notice and fix it quickly" rather than "have graceful handling that lets unintended changes slip through unnoticed"
@acdunham when you combine this into your one, can you add an assertion that the value within `count` is an integer, as expected? That way we will notice in future with a big fat error log if we break it.